23_administration/991_monitoring.txt

```diff
@@ -0,0 +1,86 @@
+=== Monitoring
+
+This section will give recommendations for:
+
+* Monitoring Rudder itself (besides standard monitoring)
+* Monitoring the state of your configuration management
+
+==== Monitoring Rudder itself
+
+===== Monitoring a Node
+
+The monitoring of a node mainly consists in checking that the Node can speak with
+its policy server, and that the agent is run regularly.
+
+A good place to start is to check the content of the last run log file.
+It can be found in '/var/rudder/cfengine-community/output/previous'.
+
+You can search lines containing 'FATAL:', 'Fatal :', or 'Could not get an updated configuration'
+with a log monitoring tool to automatically detect communication issues with the policy server.
+
+To get the last run time, you can lookup the modification date of '/var/rudder/cfengine-community/output/previous'.
+
+===== Monitoring a Server
+
+You can use use regular API calls to check the server is running and has access to its data.
```

Comment:
I added the "/rules" call. It is probably not ideal but the "/status" call is not publicly available for now.